### ISSUE TITLE
[Exception][PyCharm 2023.2.2] java.lang.RuntimeException: Cannot invoke (class=CodiumPrePRPanel$$Lambda$6389/0x000000...

### ISSUE THREAD

**Issue by saifjarboui** [Codium-ai/codiumai-jetbrains-release#128]
# User Description
_User did not enter any detailed information._

# Stack Trace
```
java.lang.RuntimeException: Cannot invoke (class=CodiumPrePRPanel$$Lambda$6389/0x0000000102587098, method=repositoryChanged, topic=GitRepository change)
	at com.intellij.util.messages.impl.MessageBusImplKt.invokeListener(MessageBusImpl.kt:676)
	at com.intellij.util.messages.impl.MessageBusImplKt.deliverMessage(MessageBusImpl.kt:422)
	at com.intellij.util.messages.impl.MessageBusImplKt.pumpWaiting(MessageBusImpl.kt:401)
	at com.intellij.util.messages.impl.MessageBusImplKt.access$pumpWaiting(MessageBusImpl.kt:1)
	at com.intellij.util.messages.impl.MessagePublisher.invoke(MessageBusImpl.kt:460)
	at jdk.proxy8/jdk.proxy8.$Proxy154.repositoryChanged(Unknown Source)
	at git4idea.repo.GitRepositoryManager.lambda$notifyListenersAsync$2(GitRepositoryManager.java:84)
	at com.intellij.util.concurrency.ContextRunnable.run(ContextRunnable.java:24)
	at com.intellij.util.concurrency.BoundedTaskExecutor.doRun(BoundedTaskExecutor.java:249)
	at com.intellij.util.concurrency.BoundedTaskExecutor.access$200(BoundedTaskExecutor.java:31)
	at com.intellij.util.concurrency.BoundedTaskExecutor$1.executeFirstTaskAndHelpQueue(BoundedTaskExecutor.java:227)
	at com.intellij.util.ConcurrencyUtil.runUnderThreadName(ConcurrencyUtil.java:218)
	at com.intellij.util.concurrency.BoundedTaskExecutor$1.run(BoundedTaskExecutor.java:215)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:702)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:699)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1.run(Executors.java:699)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.lang.ArrayIndexOutOfBoundsException: Index 397 out of bounds for length 0
	at com.intellij.ui.components.WideSelectionListUI.updateLayoutState(WideSelectionListUI.java:207)
	at java.desktop/javax.swing.plaf.basic.BasicListUI.maybeUpdateLayoutState(BasicListUI.java:1394)
	at java.desktop/javax.swing.plaf.basic.BasicListUI$Handler.valueChanged(BasicListUI.java:2815)
	at java.desktop/javax.swing.DefaultListSelectionModel.fireValueChanged(DefaultListSelectionModel.java:224)
	at java.desktop/javax.swing.DefaultListSelectionModel.fireValueChanged(DefaultListSelectionModel.java:204)
	at java.desktop/javax.swing.DefaultListSelectionModel.fireValueChanged(DefaultListSelectionModel.java:251)
	at java.desktop/javax.swing.DefaultListSelectionModel.insertIndexInterval(DefaultListSelectionModel.java:675)
	at java.desktop/javax.swing.plaf.basic.BasicListUI$Handler.intervalAdded(BasicListUI.java:2773)
	at java.desktop/javax.swing.AbstractListModel.fireIntervalAdded(AbstractListModel.java:157)
	at java.desktop/javax.swing.DefaultComboBoxModel.addElement(DefaultComboBoxModel.java:130)
	at java.desktop/javax.swing.JComboBox.addItem(JComboBox.java:746)
	at ai.codium.ui.prePR.CodiumPrePRPanel.setBranches(CodiumPrePRPanel.kt:78)
	at ai.codium.ui.prePR.CodiumPrePRPanel.setRepositoryListener$lambda$3(CodiumPrePRPanel.kt:112)
	at com.intellij.util.messages.impl.MessageBusImplKt.invokeMethod(MessageBusImpl.kt:699)
	at com.intellij.util.messages.impl.MessageBusImplKt.invokeListener(MessageBusImpl.kt:659)
	... 19 more

```

# Other Information
```
=== About ===
Build version: PyCharm 2023.2.2 Build: #PC-232.9921.89 October 1, 2023
Theme: Visual Studio Code Dark Plus
JRE: 17.0.8+7-b1000.22, JetBrains s.r.o.
JVM: 17.0.8+7-b1000.22, OpenJDK 64-Bit Server VM, JetBrains s.r.o.
Operating System: Windows 10.0 (amd64)

=== Plugins ===
Custom plugins: [Visual Studio Code Dark Plus Theme (2.8), CodiumAI - Integrity Agent powered by GPT-3.5&4 (0.6.13), Atom Material Icons (93.1.0), Requirements (2022.4.1), Ini (232.9921.89), Rainbow Brackets (2023.3.6), PowerShell (2.2.0), Ideolog (222.1.0.0), Batch Scripts Support (1.0.13), Hyper RobotFramework Support (0.1.12), Key Promoter X (2023.2.0)]
Disabled plugins:[Json Helper (1.3.0), Qodana (232.9921.89), Code With Me (232.9921.89), IntelliBot #patched (0.16.191.8026), Tabnine: AI Code Completion & Chat in Java JS/TS Python & More (1.8.0), Robot Framework Language Server (1.11.0)]
```

┆Issue is synchronized with this [Trello card](https://trello.com/c/s4XFRxp3) by [Unito](https://www.unito.io)

**Comment by aldanchenko** [comment]
Hi @saifjarboui,

Thanks for reporting!

**Comment by ArtsiomCh** [comment]
Fixed in 0.6.15
Thank you again for reporting!
